.github/workflows/test.yaml

```diff
@@ -251,6 +251,22 @@ jobs:
           # Stop local network services
           pkill hc-run-local
 
+      # - name: Smoke test - kitsune
+      #   run: |
+      #     set -x
+
+      #     # Start local bootstrap server
+      #     nix develop .#ci -c bash -c "kitsune2-bootstrap-srv &"
```

Comment:
If multiple agents should be tested with a local bootstrap like this, the bootstrap needs to be updated to accept a port.